src/main/java/frc/robot/Constants.java

```diff
@@ -115,7 +115,9 @@
   public static final class ModuleConstants {
     public static final double kMaxModuleAngularSpeedRadiansPerSecond = 0.5 * Math.PI;
     public static final double kMaxModuleAngularAccelerationRadiansPerSecondSquared = 0.5 * Math.PI;
-    public static final double kPModuleTurningController = 4.3;
+    public static double kPModuleTurningController = 4;
```

Comment:
@Grig84 Are these correct? If they are, please make them the defaults in the TunableNumber initialization because as of now they are never used

Reply:
They are numbers we were testing. Our testing was cut short, so we still need to finish testing it.

Reply:
Alright. For reference, when we were testing auto driving at MURA it went really straight, but would initially move to the side because the modules hadn't fully rotated yet. This would be corrected by the end of the trajectory, so it looks like odometry is very accurate. This suggests that we have good offsets, just a bad PID loop.